src/runtime/entitlements-manager.js

```diff
@@ -383,11 +385,13 @@ export class EntitlementsManager {
         decryptedDocumentKey
       );
       if (entitlements) {
+        this.saveSwgUserToken_(swgUserToken);
         return entitlements;
       }
     } else {
       const plainEntitlements = json['entitlements'];
       if (plainEntitlements) {
+        this.saveSwgUserToken_(json['swgUserToken']);
```

Comment:
You can use your local swgUserToken here instead of reading from the json object again.

Reply:
Good catch! I forgot to change it back after debugging